jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfiguration.kt

```diff
@@ -289,16 +289,21 @@ class LocalLambdaRunConfiguration(project: Project, factory: ConfigurationFactor
         val handler = tryOrNull { function.handler() }
             ?: throw RuntimeConfigurationError(message("lambda.run_configuration.no_handler_specified"))
 
-        val runtime = tryOrNull { Runtime.fromValue(function.runtime()).validOrNull }
-            ?: throw RuntimeConfigurationError(message("lambda.run_configuration.no_runtime_specified"))
+        val runtime = Runtime.fromValue(function.runtime()).validOrNull.validateSupportedRuntime()
```

Comment:
similar thing here and in `resolveLambdaInfo`, would be nice to tell the user if they entered a runtime that isn't modeled in the SDK

Reply:
Make sense, will improve them and post new revision

Reply:
We pass through their value now